cmd/cluster-proportional-autoscaler/options/options.go

```diff
@@ -121,4 +122,5 @@ func (c *AutoScalerConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&c.PollPeriodSeconds, "poll-period-seconds", c.PollPeriodSeconds, "The time, in seconds, to check cluster status and perform autoscale.")
 	fs.BoolVar(&c.PrintVer, "version", c.PrintVer, "Print the version and exit.")
 	fs.Var(&c.DefaultParams, "default-params", "Default parameters(JSON format) for auto-scaling. Will create/re-create a ConfigMap with this default params if ConfigMap is not present.")
+	fs.StringVar(&c.NodeLabels, "nodelabels", c.NodeLabels, "NodeLabels for filtering nodes by label selectors.")
```

Comment:
Would be great to make the usage more clear --- as I understand this is comma separated list, right?
```
--nodelabels=label1,label2
```
Or with value:
```
--nodelabels=label1=value1,label2=value2
```
Or invert
```
--nodelabels=label1!=value2
```

(Don't actual know how `LabelSelector` works nor saying we should support all, just trying to understand the usage.)

Reply:
The usage is comma separated list of key value like you said.

 In Overview section there is the flag explanation:
```
...
--nodelabels=: comma-separated list of key=value labels for filtered search of nodes and cpus.
```
But you are right, is needed a more detailed example because  this cases are accepted:

```
--nodelabels=label1,label2
```
Or
```
--nodelabels=label1=value1,label2=value2
```
So I can add it as an example in README.

Reply:
Can I add it as a new commit and then have 2 commits or is needed to group all changes in one single commit?

Reply:
Thanks for the explanation. In fact the overview section is supposed to be the same as what we put in flag's help message, so would be great to update here as well. Maybe mentioning just the most common use case would be enough.

And a single commit would be ideal given they have the same purpose, thanks.